### PR TITLE
Connect grafana to openshift-monitoring Prometheus

### PR DESCRIPTION
**Pull request by HumairAK**
Resolves: https://github.com/operate-first/SRE/issues/295 

Current RBAC was not sufficient, had to add cluster view role.

### CODE REVIEW

File: kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        type: prometheus
+        url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
```
Comment (4n4nd) [inline kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml:21]: do we want to connect to the openshift-monitoring Thanos instance and not the Prometheus instance?

Comment (HumairAK) [reply kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml:21]: I'm not sure, the connected issue stated `openshift-monitoring prometheus` 
@tumido ?

Comment (tumido) [reply kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml:21]: The blog inked in the issue was forwarded to me as "the proper" solution. It suggests to connect to thanos querier. I think this is correct solution... Let's try it this way. :slightly_smiling_face:

Comment (tumido) [reply kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml:21]: We should also track https://github.com/integr8ly/grafana-operator/issues/189 to make the token loading dynamic from secret.

**Review by tumido — COMMENTED** [review]
/approve
/lgtm

File: kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml (hunk 1 above)
Comment (HumairAK) [reply kfdefs/overlays/moc/zero/opf-monitoring/datasource/opf-openshift-monitoring-grafanadatasource.enc.yaml:21]: good call, I've subscribed to that thread

**Pull request merged**